platform/ios/src/MGLCameraChange.h

```diff
@@ -0,0 +1,35 @@
+#import "MGLFoundation.h"
+
+typedef NS_OPTIONS(NSUInteger, MGLCameraChangeReason)
```

Comment:
This type needs documentation. jazzy is configured to hide any symbol that lacks documentation, so none of the values below would appear in the docset.